### PR TITLE
Fixed manual (local) building

### PR DESCRIPTION
**Pull request by anntnzrb**
This introduces major modifications, please perform a CI build to ensure everything is working as intended.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -37,10 +37,10 @@ jobs:
       - name: Build ISO
         run: |
           # Write out the channels file so it can be included
-          guix describe -f channels > channels.scm
+          guix time-machine -C './guix/base-channels.scm' -- describe -f channels > './guix/channels.scm'
```
Comment (daviwil) [inline .github/workflows/build.yaml:28]: Now that we're doing this, maybe we should remove the channels configuration from the `Install Guix` step above?  Better to have the base channel list only in one place.

**Review by daviwil — APPROVED** [review]
Looks good so far!  If you can remove the duplicate channel config, I'll merge this and we can see how it goes.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,2 @@
-channels.scm
-guix-installer.iso
\ No newline at end of file
+*.iso
+./guix/channels.scm
```
Comment (anntnzrb) [inline .gitignore:2]: > Looks good so far! If you can remove the duplicate channel config, I'll merge this and we can see how it goes.

In the last force-push I manually deleted this file (i guess this is what you mean by duped?)

I also don't understand why the `.gitignore` won't work, the file still gets tracked anyway...

File: .github/workflows/build.yaml (hunk 1 above)
Comment (anntnzrb) [reply .github/workflows/build.yaml:28]: Indeed, it's not needed at all now.

Can you confirm these are the only changes I need to perform? I have no way to test the syntax and the actual workflow so I'm trying to be careful.

```diff
24,35d23
<         with:
<           channels: |
<             (cons* (channel
<                     (name 'nonguix)
<                     (url "https://gitlab.com/nonguix/nonguix")
<                     ;; Enable signature verification:
<                     (introduction
<                      (make-channel-introduction
<                       "897c1a470da759236cc11798f4e0a5f7d4d59fbc"
<                       (openpgp-fingerprint
<                         "2A39 3FFF 68F4 EF7A 3D29  12AF 6F51 20A0 22FB B2D5"))))
<                     %default-channels)
```

File: .gitignore (hunk 2 above)
Comment (daviwil) [reply .gitignore:2]: Nah, I'm referring to the channel config that's written literally in `build.yaml` in the Guix installation step.

Comment (anntnzrb) [reply .gitignore:2]: Alright, all good then. I've answered on that specifically already, above.

File: .github/workflows/build.yaml (hunk 1 above)
Comment (daviwil) [reply .github/workflows/build.yaml:28]: Yep, that's it!

Comment (anntnzrb) [reply .github/workflows/build.yaml:28]: > Yep, that's it!

Changes have been commited.

**Comment by daviwil** [comment]
Merging this, let's see what happens!

**Pull request merged**